DataDefinitions/FactionState.cs

```diff
@@ -22,22 +22,37 @@ static FactionState()
             // Lower tier states can cancel higher tier states from the same category.
 
             None = new FactionState("None");
-            var Retreat = new FactionState("Retreat");              // Movement state
-            var War = new FactionState("War");                      // Conflict state
-            var Lockdown = new FactionState("Lockdown");            // Security state
+            var Boom = new FactionState("Boom");                    // Economic state
+            var Bust = new FactionState("Bust");                    // Economic state
             var CivilLiberty = new FactionState("CivilLiberty");    // Security state
             var CivilUnrest = new FactionState("CivilUnrest");      // Security state
             var CivilWar = new FactionState("CivilWar");            // Conflict state
-            var Boom = new FactionState("Boom");                    // Economic state
+            var Election = new FactionState("Election");            // Conflict state
```

Comment:
Comment is wrong. This is why I dislike redundant comments.

Reply:
I disagree. Election is considered to be a conflict state. https://support.frontier.co.uk/kb/faq.php?id=470

Example:
```json
{ "timestamp":"2020-04-07T17:37:39Z", "event":"Location", "Docked":false, "StarSystem":"Borann", "SystemAddress":5070611621289, "StarPos":[123.03125,-0.25000,2.84375], "SystemAllegiance":"Federation", "SystemEconomy":"$economy_Military;", "SystemEconomy_Localised":"Military", "SystemSecondEconomy":"$economy_Refinery;", "SystemSecondEconomy_Localised":"Refinery", "SystemGovernment":"$government_Corporate;", "SystemGovernment_Localised":"Corporate", "SystemSecurity":"$SYSTEM_SECURITY_low;", "SystemSecurity_Localised":"Low Security", "Population":26868, "Body":"Borann A", "BodyID":1, "BodyType":"Star", "Factions":[ { "Name":"Caeritis Purple Fortune Ind", "FactionState":"None", "Government":"Corporate", "Influence":0.125000, "Allegiance":"Federation", "Happiness":"$Faction_HappinessBand2;", "Happiness_Localised":"Happy", "MyReputation":-1.000000, "RecoveringStates":[ { "State":"PublicHoliday", "Trend":0 } ] }, { "Name":"Grudi Democrats", "FactionState":"None", "Government":"Democracy", "Influence":0.067000, "Allegiance":"Independent", "Happiness":"$Faction_HappinessBand2;", "Happiness_Localised":"Happy", "MyReputation":0.000000 }, { "Name":"Borann Industries", "FactionState":"Boom", "Government":"Corporate", "Influence":0.518000, "Allegiance":"Federation", "Happiness":"$Faction_HappinessBand2;", "Happiness_Localised":"Happy", "MyReputation":22.301201, "ActiveStates":[ { "State":"Boom" } ] }, { "Name":"Partnership of Irula", "FactionState":"None", "Government":"Confederacy", "Influence":0.086000, "Allegiance":"Federation", "Happiness":"$Faction_HappinessBand2;", "Happiness_Localised":"Happy", "MyReputation":0.000000 }, { "Name":"Partnership of Zochna", "FactionState":"Election", "Government":"Confederacy", "Influence":0.097000, "Allegiance":"Federation", "Happiness":"$Faction_HappinessBand3;", "Happiness_Localised":"Discontented", "MyReputation":0.000000, "RecoveringStates":[ { "State":"Blight", "Trend":0 } ], "ActiveStates":[ { "State":"Famine" }, { "State":"CivilUnrest" }, { "State":"Election" } ] }, { "Name":"Friends of Borann", "FactionState":"Election", "Government":"Cooperative", "Influence":0.097000, "Allegiance":"Independent", "Happiness":"$Faction_HappinessBand2;", "Happiness_Localised":"Happy", "MyReputation":0.000000, "ActiveStates":[ { "State":"Election" } ] }, { "Name":"Borann Camorra", "FactionState":"CivilUnrest", "Government":"Anarchy", "Influence":0.010000, "Allegiance":"Independent", "Happiness":"$Faction_HappinessBand3;", "Happiness_Localised":"Discontented", "MyReputation":0.000000, "ActiveStates":[ { "State":"Bust" }, { "State":"CivilUnrest" } ] } ], "SystemFaction":{ "Name":"Borann Industries", "FactionState":"Boom" }, "Conflicts":[ { "WarType":"election", "Status":"active", "Faction1":{ "Name":"Partnership of Zochna", "Stake":"Moorcock Installation", "WonDays":0 }, "Faction2":{ "Name":"Friends of Borann", "Stake":"", "WonDays":0 } } ] }
```

Here's the conflict separated from the rest:
```
"Conflicts":[ { "WarType":"election", "Status":"active", "Faction1":{ "Name":"Partnership of Zochna", "Stake":"Moorcock Installation", "WonDays":0 }, "Faction2":{ "Name":"Friends of Borann", "Stake":"", "WonDays":0 } } ]
```